news/no-bare-except.rst

```diff
@@ -0,0 +1,12 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+Bare ``except:`` was replaced with ``except Exception`` to prevent accidentally catching utility exceptions such as KeyboardInterrupt, which caused unexpected problems like printing out the raw $PROMPT string.
```

Comment:
Can you make this a bullet point please, so it will render properly when merged with the other news. Thanks!